extras/gizmo/axis-shapes.js

```diff
@@ -106,10 +110,24 @@ class AxisShape {
         this._rotation = options.rotation ?? new Vec3();
         this._scale = options.scale ?? new Vec3(1, 1, 1);
 
+        this._disabled = options.disabled ?? false;
+
         this._layers = options.layers ?? this._layers;
 
-        this._defaultColor = options.defaultColor ?? Color.BLACK;
-        this._hoverColor = options.hoverColor ?? Color.WHITE;
+        this._defaultMaterial = options.defaultMaterial ?? Color.BLACK;
+        this._hoverMaterial = options.hoverMaterial ?? Color.WHITE;
+        this._disabledMaterial = options.disabledMaterial ?? Color.GRAY;
+    }
+
+    set disabled(value) {
+        for (let i = 0; i < this.meshInstances.length; i++) {
+            this.meshInstances[i].material = value ? this._disabledMaterial : this._defaultMaterial;
```

Comment:
so here you assign `Color.Black` as a material to mesh instance?

Reply:
Misnaming error 👍